lib/pf/Switch.pm

```diff
@@ -3550,6 +3551,39 @@ sub getCiscoAvPairAttribute {
     return ;
 }
 
+=head2 generateACLFromTemplate
+
+Generate an ACL from a template
+
+=cut
+
+sub generateACLFromTemplate {
+    my ($self, $t, $allow, $proto, $src_host, $src_port, $dst_host, $dst_port) = @_;
+    my $acl = pf::mini_template->new($t)->process({
+        allow => $allow,
+        proto => $proto,
+        src_host => $src_host // "",
+        src_port => $src_port // "",
+        dst_host => $dst_host // "",
+        dst_port => $dst_port // "",
+    });
```

Comment:
In order to make this future proof.
I would suggest passing in a hash with all the args already.
```perl
    my ($self, $t, $args) = @_
    my $acl = pf::mini_template->new($t)->process($args);
```
So if we ever need to add more args in the future it will be easy.